src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentProjectsFinder.cs

```diff
@@ -128,110 +68,89 @@ private static ImmutableArray<Project> GetProjects(Solution solution, ImmutableA
         /// Dependent projects computed in stage (1) are cached to avoid recomputation.
         /// </summary>
         private static async Task<ImmutableArray<Project>> GetDependentProjectsWorkerAsync(
-            this ISymbol symbol,
-            Solution solution,
-            CancellationToken cancellationToken)
+            Solution solution, ISymbol symbol, CancellationToken cancellationToken)
         {
             cancellationToken.ThrowIfCancellationRequested();
 
-            // Find the assembly that this symbol comes from.  (Could be a metadata or source
-            // assembly).
-            symbol = symbol.OriginalDefinition;
-            var containingAssembly = symbol.ContainingAssembly;
-            if (containingAssembly == null)
+            var assemblyAndSourceProject = GetAssemblyAndSourceProject(solution, symbol, cancellationToken);
```

Comment:
we always pass this pair around. And tehy represent the same concept "the origination informatino for the symbol".  so i just made it a tuple to make that clear and to keep them always paired together consistently.